example/multiNodeExample/storeValueFromNode2.js

```diff
@@ -71,35 +67,58 @@ const getValue = (url, privateFrom, privateFor, privateKey) => {
     });
 };
 
-const getValueFromNode1 = () => {
+const getValueFromNode1 = address => {
   return getValue(
     pantheon.node1.url,
+    address,
     orion.node1.publicKey,
     [orion.node2.publicKey],
     pantheon.node1.privateKey
   );
 };
 
-const getValueFromNode2 = () => {
+const getValueFromNode2 = address => {
   return getValue(
     pantheon.node2.url,
+    address,
     orion.node2.publicKey,
     [orion.node1.publicKey],
     pantheon.node2.privateKey
   );
 };
 
-const getValueFromNode3 = () => {
+const getValueFromNode3 = address => {
   return getValue(
     pantheon.node3.url,
+    address,
     orion.node3.publicKey,
     [orion.node1.publicKey],
     pantheon.node3.privateKey
   );
 };
 
-storeValueFromNode2(42)
-  .then(getValueFromNode1)
-  .then(getValueFromNode2)
-  .then(getValueFromNode3)
-  .catch(console.log);
+module.exports = {
+  storeValueFromNode2,
+  getValueFromNode1,
+  getValueFromNode2,
+  getValueFromNode3
+};
+
+if (require.main === module) {
+  if (!process.env.CONTRACT_ADDRESS) {
+    throw Error("You need to export CONTRACT_ADDRESS=");
```

Comment:
nit:  Consider something like "You need to export the following variable in your shell environment: CONTRACT_ADDRESS="